controllers/warehouse-controller.js

```diff
@@ -9,6 +9,27 @@ const index = async (_req, res) => {
   }
 };
 
+//this is to find a single warehouse
+const findWarehouse = async (req, res) => {
+  try {
+    const warehousesFound = await knex("warehouses").where({
+      id: req.params.id,
+    });
+    if (warehousesFound.length === 0) {
+      return res.status(404).json({
+        message: `warehouse with ID ${req.params.id} not found`,
```

Comment:
Glad this is being handled here but on the UI when I enter incorrect warehouse id in the url such as http://localhost:3000/warehouses/90000/inventories

I get a UI with all undefined. I should be getting an error modal and redirected to the appropriate page such as list of warehouses
![image](https://github.com/ryn-ngn/instock-group-2-server/assets/8683116/0ddfb67d-4041-42c4-a305-6c965c7a6a0d)